_episodes/12-export-transformation.md

```diff
@@ -1,16 +1,24 @@
 ---
-title: "Transformations - Exporting"
+title: "Exporting your revised data"
 teaching: 5
 exercises: 0
 questions:
-- "How do I export transformed data?"
+- "How do I export data from OpenRefine?"
 objectives:
 - "Explain how to export data in different formats from OpenRefine"
 keypoints:
 - "You can export your data in a variety of formats"
 ---
 
+## Note about OpenRefine
+All the edits you make to your data using OpenRefine are being stored inside the OpenRefine program and are not being saved to your original file. That's why OpenRefine uses the terms "import" and "export" to talk about moving your data in and out of the OpenRefine interface. Therefore, in order to save your work into a file format you can view in other programs or share with others, you need to export your data.
+
 ## Exporting data
 Once you have finished working with a data set in OpenRefine you may wish to export it. The export options are accessed through the ```Export``` button at the top right of the OpenRefine interface.
 
 Export formats support include HTML, Excel and comma- and tab-separated value (csv and tsv). You can also write a custom export, selecting to export specific fields, adding a header or footer and specifying the exact format.
+
+## Exporting a portion of your data
+You can also export a portion of your data by using facets or filters to select a portion of your data. With only those rows selected, you can select the export format and your resulting file will only include the select rows. 
+
+*Note well:* It's easy to export only a portion of your data by accident, so make sure you look at the top left to ensure all rows are being displayed when you want to do a full export.  
```

Comment:
If you use the custom tabular export you also get the option to override the current facets - but that may be too much detail here? Happy with this change as you've written it unless you feel this detail is worth adding